From 98721be1c1762adec9130f15263d98e575e290ae Mon Sep 17 00:00:00 2001
From: Alexandre Belloni <alexandre.belloni@free-electrons.com>
Date: Thu, 8 Aug 2013 20:14:51 +0200
Subject: [PATCH] imx-bootlets: Fix paths used during boot stream generation

This will be run during image generation thus this won't use a
complete path to allow for relocation.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
---
 barebox_ivt.bd | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/barebox_ivt.bd b/barebox_ivt.bd
index 0c67e9c..c603f9e 100644
--- a/barebox_ivt.bd
+++ b/barebox_ivt.bd
@@ -1,9 +1,9 @@
 // STMP378x ROM command script to load and run U-Boot
 
 sources {
-	power_prep="./power_prep/power_prep";
-	sdram_prep="./boot_prep/boot_prep";
-	barebox="./barebox";
+	power_prep="imx-bootlets-power_prep-@MACHINE@";
+	sdram_prep="imx-bootlets-boot_prep-@MACHINE@";
+	barebox="barebox-@MACHINE@.bin";
 }
 
 section (0) {
-- 
1.8.1.2

